layouts/_default/list.html

```diff
@@ -5,35 +5,28 @@
     <title>{{ block "title" . }}{{ .Site.Title }}{{ end }}</title>
     {{ partial "css.html" . }}
   </head>
-<body>
-  <!-- Preloader -->
-  <div id="preloader">
-    <div id="status"></div>
-  </div>
+  <body>
+    <!-- Preloader -->
+    <div id="preloader">
+      <div id="status"></div>
+    </div>
 
     {{ partial "single/single.html" . }}
-
-<div class="container">
-    {{ range .Pages }}
-        <div class="tile">
-            <a href="{{.Permalink}}">{{.Date.Format "2006-01-02"}} | {{.Title}}</a>
-        </div>
-    {{ end }}
-</div>
-
-<br>
+    {{ $posts := (where .Site.RegularPages "Section" "blog") }}
+    {{ partial "news.html" $posts }}
```

Comment:
I pulled out the `range` of posts from this partial into a `news.html` partial so we can feed it all blog posts on the News page and limit what we feed the partial on the Homepage.

Reply:
Cool, that is what I wanted.